atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueService.java

```diff
@@ -849,73 +920,77 @@ public void delete(final TableReference tableRef, final Multimap<Cell, Long> key
         bindQuery.append("LIMIT " + batchHint);
         final String getLastRowQuery = "SELECT * FROM " + getFullTableName(tableRef) + " WHERE "
                 + CassandraConstants.ROW_NAME + " = ?";
-        return ClosableIterators.wrap(new AbstractPagingIterable<RowResult<U>, TokenBackedBasicResultsPage<RowResult<U>, byte[]>>() {
-            @Override
-            protected TokenBackedBasicResultsPage<RowResult<U>, byte[]> getFirstPage()
-                    throws Exception {
-                return getPage(rangeRequest.getStartInclusive());
-            }
+        return ClosableIterators.wrap(
+                new AbstractPagingIterable<RowResult<U>, TokenBackedBasicResultsPage<RowResult<U>, byte[]>>() {
+                @Override
+                protected TokenBackedBasicResultsPage<RowResult<U>, byte[]> getFirstPage()
```

Comment:
Not sure exactly what the style guide says, but shouldn't this be more indented than the statement two lines above?